README.md

```diff
@@ -48,48 +48,30 @@ buildbot start .
 ```
 ### Deploy build box Worker Buildbot
 Prerequisites:
-- CentOS v7.3
+- CentOS v6.9 / v7.3
 - Python v3.6.x
 
-Needed packages (dependencies):
-```bash
-sudo yum install libpciaccess libpciaccess-devel opencl-headers
-
-#Enable epel repository
-sudo yum install gtest gtest-devel ocl-icd ocl-icd-devel
-
-#Compile OpenCL from these cources: https://github.com/intel/compute-runtime/releases/tag/2018ww18-010782
-#By using these instructions: https://github.com/intel/compute-runtime/blob/master/documentation/BUILD_Centos.md
-sudo rpm -ihv intel-opencl-1.0-0.x86_64-igdrcl.rpm
+To set up environment for MediaSDK and install all dependencies do:
+- [Build Media SDK on CentOS](https://github.com/Intel-Media-SDK/MediaSDK/wiki/Build-Media-SDK-on-CentOS)
```

Comment:
Is it ok in context of CI deploying `export PKG_CONFIG_PATH=${PKG_CONFIG_PATH}:/usr/local/lib/pkgconfig`?
(Follow this link https://github.com/Intel-Media-SDK/MediaSDK/wiki/Build-Media-SDK-on-CentOS)

Reply:
Better:
export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH
But it should not be a part of deploy. This should be set before building something

Reply:
ok